README.adoc

```diff
@@ -323,6 +323,31 @@ foo&. bar
 foo&.bar
 ----
 
+=== Safe navigation
+
+Avoid long chains of `&.` replace with `.`, if safe navigator operator introduce unnecessary conditional logic.
+Consider other approaches such as delegation.
+
+If users are guaranteed to have an address and addresses are guaranteed to have a zip code:
+
+[source, thoughtbot]
```

Comment:
```suggestion
[source,ruby]
```
According to the [AsciiDoc documentation](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/), the second argument is the language of the source.

No wonder "thoughtbot" here breaks code highlighting:
![image](https://user-images.githubusercontent.com/6916/209580738-28d81f91-5433-4402-a090-9fecb7f5b09e.png)